### ISSUE TITLE
Voice command button causes Google to crash

### ISSUE THREAD

**Issue by magd1978** [openhab/openhab-android#1384]
The first, and sometimes the second time you press the voice command button, an error occurs in Google services.  Normally the third time it is pressed works well.  The Android version is 7.1.1 and the Google services are 9.91.6.21 weapons.

See log:
````
--------- beginning of events
06-03 20:30:39.044  3825  3825 I am_on_resume_called: [0,org.openhab.habdroid.ui.MainActivity,LAUNCH_ACTIVITY]
06-03 20:30:47.472  3825  3825 I am_on_paused_called: [0,org.openhab.habdroid.ui.MainActivity,handlePauseActivity]
06-03 20:30:47.644  3825  3825 I am_on_stop_called: [0,org.openhab.habdroid.ui.MainActivity,destroy]
--------- beginning of main
06-03 20:41:57.266  9462  9462 I art     : Starting a blocking GC AddRemoveAppImageSpace
06-03 20:41:57.269  9462  9462 W System  : ClassLoader referenced unknown path: /data/app/org.openhab.habdroid-2/lib/arm
06-03 20:41:57.332  9462  9462 I MultiDex: VM with version 2.1.0 has multidex support
06-03 20:41:57.341  9462  9462 I MultiDex: Installing application
06-03 20:41:57.341  9462  9462 I MultiDex: VM has multidex support, MultiDex support library is disabled.
06-03 20:41:57.382  9462  9462 D FirebaseApp: com.google.firebase.auth.FirebaseAuth is not linked. Skipping initialization.
06-03 20:41:57.384  9462  9462 D FirebaseApp: com.google.firebase.crash.FirebaseCrash is not linked. Skipping initialization.
06-03 20:41:57.385  9462  9462 D FirebaseApp: com.google.android.gms.measurement.AppMeasurement is not linked. Skipping initialization.
06-03 20:41:57.385  9462  9462 I FirebaseInitProvider: FirebaseApp initialization successful
06-03 20:41:57.439  9462  9462 I CrashlyticsCore: Initializing Crashlytics Core 2.7.0.33
06-03 20:41:57.448  9462  9494 W Fabric  : Could not call getId on com.google.android.gms.ads.identifier.AdvertisingIdClient$Info
06-03 20:41:57.449  9462  9494 W Fabric  : Could not call isLimitAdTrackingEnabled on com.google.android.gms.ads.identifier.AdvertisingIdClient$Info
06-03 20:41:57.453  9462  9462 I CrashlyticsInitProvider: CrashlyticsInitProvider initialization successful
06-03 20:41:57.556  9462  9462 D NetworkSecurityConfig: Using Network Security Config from resource network_security_config debugBuild: false
06-03 20:41:57.914  9462  9505 D AbstractConnection: Checking reachability of https://<openhab-local-address>/
06-03 20:41:57.925  9462  9462 D MainActivity: onCreate()
06-03 20:41:57.964  9462  9462 I art     : Rejecting re-init on previously-failed class java.lang.Class<androidx.core.view.ViewCompat$OnUnhandledKeyEventListenerWrapper>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/view/View$OnUnhandledKeyEventListener;
06-03 20:41:57.964  9462  9462 I art     :   at android.view.ViewGroup androidx.appcompat.app.AppCompatDelegateImpl.createSubDecor() (AppCompatDelegateImpl.java:637)
06-03 20:41:57.964  9462  9462 I art     :   at void androidx.appcompat.app.AppCompatDelegateImpl.ensureSubDecor() (AppCompatDelegateImpl.java:518)
06-03 20:41:57.964  9462  9462 I art     :   at void androidx.appcompat.app.AppCompatDelegateImpl.setContentView(int) (AppCompatDelegateImpl.java:466)
06-03 20:41:57.964  9462  9462 I art     :   at void androidx.appcompat.app.AppCompatActivity.setContentView(int) (AppCompatActivity.java:140)
06-03 20:41:57.964  9462  9462 I art     :   at void org.openhab.habdroid.ui.MainActivity.onCreate(android.os.Bundle) (MainActivity.java:203)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.Activity.performCreate(android.os.Bundle) (Activity.java:6720)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.Instrumentation.callActivityOnCreate(android.app.Activity, android.os.Bundle) (Instrumentation.java:1119)
06-03 20:41:57.964  9462  9462 I art     :   at android.app.Activity android.app.ActivityThread.performLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent) (ActivityThread.java:2618)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread.handleLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent, java.lang.String) (ActivityThread.java:2726)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread.-wrap12(android.app.ActivityThread, android.app.ActivityThread$ActivityClientRecord, android.content.Intent, java.lang.String) (ActivityThread.java:-1)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread$H.handleMessage(android.os.Message) (ActivityThread.java:1477)
06-03 20:41:57.964  9462  9462 I art     :   at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:102)
06-03 20:41:57.964  9462  9462 I art     :   at void android.os.Looper.loop() (Looper.java:154)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6119)
06-03 20:41:57.964  9462  9462 I art     :   at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
06-03 20:41:57.964  9462  9462 I art     :   at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:886)
06-03 20:41:57.964  9462  9462 I art     :   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:776)
06-03 20:41:57.964  9462  9462 I art     : Caused by: java.lang.ClassNotFoundException: Didn't find class "android.view.View$OnUnhandledKeyEventListener" on path: DexPathList[[zip file "/data/app/org.openhab.habdroid-2/base.apk"],nativeLibraryDirectories=[/data/app/org.openhab.habdroid-2/lib/arm, /system/lib, /vendor/lib]]
06-03 20:41:57.964  9462  9462 I art     :   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:56)
06-03 20:41:57.964  9462  9462 I art     :   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:380)
06-03 20:41:57.964  9462  9462 I art     :   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
06-03 20:41:57.964  9462  9462 I art     :   at android.view.ViewGroup androidx.appcompat.app.AppCompatDelegateImpl.createSubDecor() (AppCompatDelegateImpl.java:637)
06-03 20:41:57.964  9462  9462 I art     :   at void androidx.appcompat.app.AppCompatDelegateImpl.ensureSubDecor() (AppCompatDelegateImpl.java:518)
06-03 20:41:57.964  9462  9462 I art     :   at void androidx.appcompat.app.AppCompatDelegateImpl.setContentView(int) (AppCompatDelegateImpl.java:466)
06-03 20:41:57.964  9462  9462 I art     :   at void androidx.appcompat.app.AppCompatActivity.setContentView(int) (AppCompatActivity.java:140)
06-03 20:41:57.964  9462  9462 I art     :   at void org.openhab.habdroid.ui.MainActivity.onCreate(android.os.Bundle) (MainActivity.java:203)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.Activity.performCreate(android.os.Bundle) (Activity.java:6720)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.Instrumentation.callActivityOnCreate(android.app.Activity, android.os.Bundle) (Instrumentation.java:1119)
06-03 20:41:57.964  9462  9462 I art     :   at android.app.Activity android.app.ActivityThread.performLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent) (ActivityThread.java:2618)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread.handleLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent, java.lang.String) (ActivityThread.java:2726)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread.-wrap12(android.app.ActivityThread, android.app.ActivityThread$ActivityClientRecord, android.content.Intent, java.lang.String) (ActivityThread.java:-1)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread$H.handleMessage(android.os.Message) (ActivityThread.java:1477)
06-03 20:41:57.964  9462  9462 I art     :   at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:102)
06-03 20:41:57.964  9462  9462 I art     :   at void android.os.Looper.loop() (Looper.java:154)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6119)
06-03 20:41:57.964  9462  9462 I art     :   at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
06-03 20:41:57.964  9462  9462 I art     :   at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:886)
06-03 20:41:57.964  9462  9462 I art     :   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:776)
06-03 20:41:57.964  9462  9462 I art     : 
06-03 20:41:57.979  9462  9462 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
06-03 20:41:57.996  9462  9462 V BoostFramework: mAcquireFunc method = public int com.qualcomm.qti.Performance.perfLockAcquire(int,int[])
06-03 20:41:57.997  9462  9462 V BoostFramework: mReleaseFunc method = public int com.qualcomm.qti.Performance.perfLockRelease()
06-03 20:41:57.997  9462  9462 V BoostFramework: mAcquireTouchFunc method = public int com.qualcomm.qti.Performance.perfLockAcquireTouch(android.view.MotionEvent,android.util.DisplayMetrics,int,int[])
06-03 20:41:57.997  9462  9462 V BoostFramework: mIOPStart method = public int com.qualcomm.qti.Performance.perfIOPrefetchStart(int,java.lang.String)
06-03 20:41:57.997  9462  9462 V BoostFramework: mIOPStop method = public int com.qualcomm.qti.Performance.perfIOPrefetchStop()
06-03 20:41:58.001  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@7b65ee2
06-03 20:41:58.001  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@e5b7273
06-03 20:41:58.001  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@1c0f530
06-03 20:41:58.001  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@cfd35a9
06-03 20:41:58.010  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@f0c2cf
06-03 20:41:58.010  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@cf8345c
06-03 20:41:58.012  9462  9462 I ViewConfigCompat: Could not find method getScaledScrollFactor() on ViewConfiguration
06-03 20:41:58.034  9462  9462 D MainActivity: Got intent: Intent { flg=0x10000000 cmp=org.openhab.habdroid/.ui.MainActivity bnds=[834,1219][1056,1344] }
06-03 20:41:58.039  9462  9462 D MainActivity: onStart()
06-03 20:41:58.039  9462  9462 D PageConnectionHolderFragment: onStart(), started false
06-03 20:41:58.040  9462  9462 D MainActivity: onAvailableConnectionChanged()
06-03 20:41:58.040  9462  9462 D ContentController: Update to connection null (message null)
06-03 20:41:58.040  9462  9462 D PageConnectionHolderFragment: updateActiveConnections: URL list [], connection null
06-03 20:41:58.041  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@e31db19
06-03 20:41:58.041  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@a5217de
06-03 20:41:58.052  9462  9462 D MainActivity: onPostCreate()
06-03 20:41:58.053  9462  9462 D MainActivity: onResume()
06-03 20:41:58.057  9462  9462 I am_on_resume_called: [0,org.openhab.habdroid.ui.MainActivity,LAUNCH_ACTIVITY]
06-03 20:41:58.094  9462  9509 I Adreno  : QUALCOMM build                   : aa39516, I97e7dfc697
06-03 20:41:58.094  9462  9509 I Adreno  : Build Date                       : 04/17/17
06-03 20:41:58.094  9462  9509 I Adreno  : OpenGL ES Shader Compiler Version: XE031.09.00.04
06-03 20:41:58.094  9462  9509 I Adreno  : Local Branch                     : 
06-03 20:41:58.094  9462  9509 I Adreno  : Remote Branch                    : quic/LA.BR.1.3.6.c2_rb2.12
06-03 20:41:58.094  9462  9509 I Adreno  : Remote Branch                    : NONE
06-03 20:41:58.094  9462  9509 I Adreno  : Reconstruct Branch               : NOTHING
06-03 20:41:58.100  9462  9509 I OpenGLRenderer: Initialized EGL, version 1.4
06-03 20:41:58.100  9462  9509 D OpenGLRenderer: Swap behavior 1
06-03 20:41:58.131  9462  9462 D MainActivity: onCreateOptionsMenu()
06-03 20:41:58.131  9462  9462 D MainActivity: onPrepareOptionsMenu()
06-03 20:41:58.824  9462  9505 D AbstractConnection: Socket connected (attempt  0)
06-03 20:41:58.824  9462  9505 D ConnectionFactory: Connecting to local URL
06-03 20:41:58.824  9462  9462 D MainActivity: onAvailableConnectionChanged()
06-03 20:41:58.829  9462  9462 D ContentController: Update to connection org.openhab.habdroid.core.connection.DefaultConnection@4685fd8 (message null)
06-03 20:41:58.829  9462  9462 D PageConnectionHolderFragment: updateActiveConnections: URL list [], connection org.openhab.habdroid.core.connection.DefaultConnection@4685fd8
06-03 20:41:58.832  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@7aa8b97
06-03 20:41:58.832  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@7efa184
06-03 20:41:58.842  9462  9462 D MainActivity: onCreateOptionsMenu()
06-03 20:41:58.842  9462  9462 D MainActivity: onPrepareOptionsMenu()
06-03 20:41:59.093  9462  9462 D ServerProperties: Server returned sitemaps: [Sitemap{name=tf, label=Tempus Fugit, link=https://<openhab-local-address>/rest/sitemaps/tf, icon=null, iconPath=icon/null, homepageLink=https://<openhab-local-address>/rest/sitemaps/tf/tf}]
06-03 20:41:59.096  9462  9462 D MainActivity: Configured sitemap is 'tf', selected Sitemap{name=tf, label=Tempus Fugit, link=https://<openhab-local-address>/rest/sitemaps/tf, icon=null, iconPath=icon/null, homepageLink=https://<openhab-local-address>/rest/sitemaps/tf/tf}
06-03 20:41:59.096  9462  9462 I MainActivity: Opening sitemap Sitemap{name=tf, label=Tempus Fugit, link=https://<openhab-local-address>/rest/sitemaps/tf, icon=null, iconPath=icon/null, homepageLink=https://<openhab-local-address>/rest/sitemaps/tf/tf}, currently selected null
06-03 20:41:59.096  9462  9462 D ContentController: Opening sitemap Sitemap{name=tf, label=Tempus Fugit, link=https://<openhab-local-address>/rest/sitemaps/tf, icon=null, iconPath=icon/null, homepageLink=https://<openhab-local-address>/rest/sitemaps/tf/tf}
06-03 20:41:59.097  9462  9462 D PageConnectionHolderFragment: updateActiveConnections: URL list [https://<openhab-local-address>/rest/sitemaps/tf/tf], connection org.openhab.habdroid.core.connection.DefaultConnection@4685fd8
06-03 20:41:59.097  9462  9462 D PageConnectionHolderFragment: Creating new handler for URL https://<openhab-local-address>/rest/sitemaps/tf/tf
06-03 20:41:59.097  9462  9462 D PageConnectionHolderFragment: Creating new SSE helper for sitemap tf, page tf
06-03 20:41:59.097  9462  9462 D PageConnectionHolderFragment: Loading data for https://<openhab-local-address>/rest/sitemaps/tf/tf, long polling false
06-03 20:41:59.103  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@c59486f
06-03 20:41:59.163  9462  9462 D MainActivity: onOptionsItemSelected()
06-03 20:41:59.272  9462  9462 D PageConnectionHolderFragment: Updated page data for URL https://<openhab-local-address>/rest/sitemaps/tf/tf (12 widgets)
06-03 20:41:59.272  9462  9462 D ContentController: Got update for URL https://<openhab-local-address>/rest/sitemaps/tf/tf, pending [https://<openhab-local-address>/rest/sitemaps/tf/tf]
06-03 20:41:59.288  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@7175cae
06-03 20:41:59.288  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@fc3aa4f
06-03 20:41:59.291  9462  9462 D WidgetListFragment: onViewCreated() https://<openhab-local-address>/rest/sitemaps/tf/tf
06-03 20:41:59.291  9462  9462 D WidgetListFragment: onActivityCreated() https://<openhab-local-address>/rest/sitemaps/tf/tf
06-03 20:41:59.292  9462  9462 D WidgetListFragment: onStart() https://<openhab-local-address>/rest/sitemaps/tf/tf
06-03 20:41:59.293  9462  9462 D PageConnectionHolderFragment: Trigger update for URL https://<openhab-local-address>/rest/sitemaps/tf/tf, force false
06-03 20:41:59.293  9462  9462 D ContentController: Got update for URL https://<openhab-local-address>/rest/sitemaps/tf/tf, pending []
06-03 20:41:59.304  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/attic?state=&format=PNG, avoidCache false
06-03 20:41:59.304  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/attic?state=&format=PNG, avoidCache false
06-03 20:41:59.311  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/firstfloor?state=&format=PNG, avoidCache false
06-03 20:41:59.312  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/firstfloor?state=&format=PNG, avoidCache false
06-03 20:41:59.317  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/groundfloor?state=&format=PNG, avoidCache false
06-03 20:41:59.318  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/groundfloor?state=&format=PNG, avoidCache false
06-03 20:41:59.323  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/terrace?state=&format=PNG, avoidCache false
06-03 20:41:59.324  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/terrace?state=&format=PNG, avoidCache false
06-03 20:41:59.334  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/temperature?state=26.54&format=PNG, avoidCache false
06-03 20:41:59.335  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/temperature?state=26.54&format=PNG, avoidCache false
06-03 20:41:59.343  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/calendar?state=2019-06-03T20%3A41%3A46.738%2B0200&format=PNG, avoidCache false
06-03 20:41:59.344  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/calendar?state=2019-06-03T20%3A41%3A46.738%2B0200&format=PNG, avoidCache false
06-03 20:41:59.356  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/settings?state=&format=PNG, avoidCache false
06-03 20:41:59.357  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/settings?state=&format=PNG, avoidCache false
06-03 20:41:59.362  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/lock?state=&format=PNG, avoidCache false
06-03 20:41:59.363  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/lock?state=&format=PNG, avoidCache false
06-03 20:42:00.404  9462  9462 D MainActivity: onCloudConnectionChanged()
06-03 20:42:00.421  9462  9539 D FcmRegistrationService: Register device at openHAB-cloud with URL: addAndroidRegistration?deviceId=9a2d1c5dfcb4c1e6&deviceModel=Bq+Aquaris+X5+Plus&regId=fIQBgMkvUG4:APA91bELay_N-4ZBqbKGFRcBoREv__EmH4vXSgKuJt9FmaWD4PEoHGrL2mtOHj6bOxe03D6EfpMzDSjFuYinX-DXWau_-VaN4DgAHC32aKMRsmNW3CGfxbMYGz0eV7S9IwJEzpq5GdpPRdmKILa1bvYV9SV1W0OyDw
06-03 20:42:01.220  9462  9539 D FcmRegistrationService: FCM reg id success
06-03 20:42:01.575  9462  9462 D MainActivity: onPause()
06-03 20:42:01.575  9462  9462 D WidgetListFragment: onPause() https://<openhab-local-address>/rest/sitemaps/tf/tf
06-03 20:42:01.577  9462  9462 I am_on_paused_called: [0,org.openhab.habdroid.ui.MainActivity,handlePauseActivity]
06-03 20:42:01.610  9462  9462 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
06-03 20:42:01.651  9462  9462 E ExtMediaPlayer-JNI: env->IsInstanceOf fails
06-03 20:42:01.651  9462  9462 E MediaPlayer-JNI: JNIMediaPlayerFactory: bIsQCMediaPlayerPresent 0
06-03 20:42:01.651  9462  9462 E ExtMediaPlayer-JNI: env->IsInstanceOf fails
06-03 20:42:01.651  9462  9462 E MediaPlayer-JNI: JNIMediaPlayerFactory: bIsQCMediaPlayerPresent 0
06-03 20:42:01.773  9462  9462 D MediaPlayer: setSubtitleAnchor in MediaPlayer
06-03 20:42:01.779  9462  9462 D Ringtone: Successfully created local player
06-03 20:42:01.796  9462  9462 D PreferencesActivity: Removing notification prefs for >= 25
06-03 20:42:01.814  9462  9462 I am_on_resume_called: [0,org.openhab.habdroid.ui.PreferencesActivity,LAUNCH_ACTIVITY]
06-03 20:42:02.058  9462  9462 D MainActivity: onSaveInstanceState()
06-03 20:42:02.064  9462  9462 D MainActivity: onStop()
06-03 20:42:02.064  9462  9462 D PageConnectionHolderFragment: onStop()
06-03 20:42:02.064  9462  9462 D PageConnectionHolderFragment: Canceling connection for URL https://<openhab-local-address>/rest/sitemaps/tf/tf
06-03 20:42:02.065  9462  9462 I am_on_stop_called: [0,org.openhab.habdroid.ui.MainActivity,handleStopActivity]
06-03 20:42:02.524  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@de7ff45
06-03 20:42:02.525  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@fece69a
06-03 20:42:03.730  9462  9462 I am_on_paused_called: [0,org.openhab.habdroid.ui.PreferencesActivity,handlePauseActivity]
06-03 20:42:03.770  9462  9462 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
06-03 20:42:03.788  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@2d8649d
06-03 20:42:03.788  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@338dd12
06-03 20:42:03.799  9462  9462 I am_on_resume_called: [0,org.openhab.habdroid.ui.LogActivity,LAUNCH_ACTIVITY]
````

**Comment by mueller-ma** [comment]
Please use the issue template:




### Actual behaviour
Describe us what happens


### Expected behaviour
Describe us what should happen

 
### Steps to reproduce
1. Describe us how to reproduce the issue
2. Use a list for it


### Can you reproduce the issue in demo mode?




### Environment data
#### Client
* Android version: 
* Device model: 
* Stock or customized ROM: 
* App version : 
* Build flavor : 
* Device language: 

#### Server
* Server version: 
* Reverse Proxy: 
* Authentication method : 

#### openHAB Cloud
* openHAB Cloud version: 
* Self hosted cloud or myopenhab.org: 
* Authentication method : 

### Logs
#### App log
```
Open the app, go to `Settings` => `View log` and insert the log here.
```

#### openHAB Server log
```
Insert your log here. If you have installed openHAB via `apt-get`, the log is located under `/var/log/openhab2/openhab.log`
```

#### Reverse Proxy log
```
Insert your reverse proxy error log here.
For nginx: `/var/log/nginx/error.log`
For Apache 2: `/var/log/apache2/error.log`
```
```
Insert your reverse proxy access log here
For nginx: `/var/log/nginx/access.log`
For Apache 2: `/var/log/apache2/access.log`
```

**Comment by mueller-ma** [comment]
Maybe duplicate of #1334

**Comment by maniac103** [comment]
If the Google app crashes as the report seems to imply, that one seems unrelated (and shouldn't we have the crash in the log in that case?)
@magd1978 So what exactly crashes? And in case it's really GMS, do you by chance have a chance to grab the global crash log after the fact, either by using adb or - on a rooted device - by an app like MatLog?

**Comment by magd1978** [comment]
### **Actual behaviour**
The first, and sometimes the second time you press the voice command button, an error occurs in Google services.

### **Expected behaviour**
No error must occour.

### **Steps to reproduce**
Pressing VoiceCommand button on openHab Android app or launcher widget.

**Can you reproduce the issue in demo mode?**
Yes. The failure occurs when you invoke Google services for speech recognition, regardless of whether or not there is a connection to an openHab server.

### **Environment data**
**Client**
Android version: 7.1.1
Device model: bq Aquaris X5 plus
Stock or customized ROM: stock
App version : 2.7.0
Build flavor :
Device language: Spanish
**Server**
Server version: not applicable
Reverse Proxy: not applicable
Authentication method : not applicable
openHAB Cloud
openHAB Cloud version: not applicable
Self hosted cloud or myopenhab.org: not applicable
Authentication method : not applicable

### **Logs**
**App log**

```
--------- beginning of events
06-03 20:30:39.044  3825  3825 I am_on_resume_called: [0,org.openhab.habdroid.ui.MainActivity,LAUNCH_ACTIVITY]
06-03 20:30:47.472  3825  3825 I am_on_paused_called: [0,org.openhab.habdroid.ui.MainActivity,handlePauseActivity]
06-03 20:30:47.644  3825  3825 I am_on_stop_called: [0,org.openhab.habdroid.ui.MainActivity,destroy]
--------- beginning of main
06-03 20:41:57.266  9462  9462 I art     : Starting a blocking GC AddRemoveAppImageSpace
06-03 20:41:57.269  9462  9462 W System  : ClassLoader referenced unknown path: /data/app/org.openhab.habdroid-2/lib/arm
06-03 20:41:57.332  9462  9462 I MultiDex: VM with version 2.1.0 has multidex support
06-03 20:41:57.341  9462  9462 I MultiDex: Installing application
06-03 20:41:57.341  9462  9462 I MultiDex: VM has multidex support, MultiDex support library is disabled.
06-03 20:41:57.382  9462  9462 D FirebaseApp: com.google.firebase.auth.FirebaseAuth is not linked. Skipping initialization.
06-03 20:41:57.384  9462  9462 D FirebaseApp: com.google.firebase.crash.FirebaseCrash is not linked. Skipping initialization.
06-03 20:41:57.385  9462  9462 D FirebaseApp: com.google.android.gms.measurement.AppMeasurement is not linked. Skipping initialization.
06-03 20:41:57.385  9462  9462 I FirebaseInitProvider: FirebaseApp initialization successful
06-03 20:41:57.439  9462  9462 I CrashlyticsCore: Initializing Crashlytics Core 2.7.0.33
06-03 20:41:57.448  9462  9494 W Fabric  : Could not call getId on com.google.android.gms.ads.identifier.AdvertisingIdClient$Info
06-03 20:41:57.449  9462  9494 W Fabric  : Could not call isLimitAdTrackingEnabled on com.google.android.gms.ads.identifier.AdvertisingIdClient$Info
06-03 20:41:57.453  9462  9462 I CrashlyticsInitProvider: CrashlyticsInitProvider initialization successful
06-03 20:41:57.556  9462  9462 D NetworkSecurityConfig: Using Network Security Config from resource network_security_config debugBuild: false
06-03 20:41:57.914  9462  9505 D AbstractConnection: Checking reachability of https://<openhab-local-address>/
06-03 20:41:57.925  9462  9462 D MainActivity: onCreate()
06-03 20:41:57.964  9462  9462 I art     : Rejecting re-init on previously-failed class java.lang.Class<androidx.core.view.ViewCompat$OnUnhandledKeyEventListenerWrapper>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/view/View$OnUnhandledKeyEventListener;
06-03 20:41:57.964  9462  9462 I art     :   at android.view.ViewGroup androidx.appcompat.app.AppCompatDelegateImpl.createSubDecor() (AppCompatDelegateImpl.java:637)
06-03 20:41:57.964  9462  9462 I art     :   at void androidx.appcompat.app.AppCompatDelegateImpl.ensureSubDecor() (AppCompatDelegateImpl.java:518)
06-03 20:41:57.964  9462  9462 I art     :   at void androidx.appcompat.app.AppCompatDelegateImpl.setContentView(int) (AppCompatDelegateImpl.java:466)
06-03 20:41:57.964  9462  9462 I art     :   at void androidx.appcompat.app.AppCompatActivity.setContentView(int) (AppCompatActivity.java:140)
06-03 20:41:57.964  9462  9462 I art     :   at void org.openhab.habdroid.ui.MainActivity.onCreate(android.os.Bundle) (MainActivity.java:203)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.Activity.performCreate(android.os.Bundle) (Activity.java:6720)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.Instrumentation.callActivityOnCreate(android.app.Activity, android.os.Bundle) (Instrumentation.java:1119)
06-03 20:41:57.964  9462  9462 I art     :   at android.app.Activity android.app.ActivityThread.performLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent) (ActivityThread.java:2618)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread.handleLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent, java.lang.String) (ActivityThread.java:2726)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread.-wrap12(android.app.ActivityThread, android.app.ActivityThread$ActivityClientRecord, android.content.Intent, java.lang.String) (ActivityThread.java:-1)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread$H.handleMessage(android.os.Message) (ActivityThread.java:1477)
06-03 20:41:57.964  9462  9462 I art     :   at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:102)
06-03 20:41:57.964  9462  9462 I art     :   at void android.os.Looper.loop() (Looper.java:154)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6119)
06-03 20:41:57.964  9462  9462 I art     :   at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
06-03 20:41:57.964  9462  9462 I art     :   at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:886)
06-03 20:41:57.964  9462  9462 I art     :   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:776)
06-03 20:41:57.964  9462  9462 I art     : Caused by: java.lang.ClassNotFoundException: Didn't find class "android.view.View$OnUnhandledKeyEventListener" on path: DexPathList[[zip file "/data/app/org.openhab.habdroid-2/base.apk"],nativeLibraryDirectories=[/data/app/org.openhab.habdroid-2/lib/arm, /system/lib, /vendor/lib]]
06-03 20:41:57.964  9462  9462 I art     :   at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:56)
06-03 20:41:57.964  9462  9462 I art     :   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:380)
06-03 20:41:57.964  9462  9462 I art     :   at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
06-03 20:41:57.964  9462  9462 I art     :   at android.view.ViewGroup androidx.appcompat.app.AppCompatDelegateImpl.createSubDecor() (AppCompatDelegateImpl.java:637)
06-03 20:41:57.964  9462  9462 I art     :   at void androidx.appcompat.app.AppCompatDelegateImpl.ensureSubDecor() (AppCompatDelegateImpl.java:518)
06-03 20:41:57.964  9462  9462 I art     :   at void androidx.appcompat.app.AppCompatDelegateImpl.setContentView(int) (AppCompatDelegateImpl.java:466)
06-03 20:41:57.964  9462  9462 I art     :   at void androidx.appcompat.app.AppCompatActivity.setContentView(int) (AppCompatActivity.java:140)
06-03 20:41:57.964  9462  9462 I art     :   at void org.openhab.habdroid.ui.MainActivity.onCreate(android.os.Bundle) (MainActivity.java:203)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.Activity.performCreate(android.os.Bundle) (Activity.java:6720)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.Instrumentation.callActivityOnCreate(android.app.Activity, android.os.Bundle) (Instrumentation.java:1119)
06-03 20:41:57.964  9462  9462 I art     :   at android.app.Activity android.app.ActivityThread.performLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent) (ActivityThread.java:2618)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread.handleLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent, java.lang.String) (ActivityThread.java:2726)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread.-wrap12(android.app.ActivityThread, android.app.ActivityThread$ActivityClientRecord, android.content.Intent, java.lang.String) (ActivityThread.java:-1)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread$H.handleMessage(android.os.Message) (ActivityThread.java:1477)
06-03 20:41:57.964  9462  9462 I art     :   at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:102)
06-03 20:41:57.964  9462  9462 I art     :   at void android.os.Looper.loop() (Looper.java:154)
06-03 20:41:57.964  9462  9462 I art     :   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6119)
06-03 20:41:57.964  9462  9462 I art     :   at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
06-03 20:41:57.964  9462  9462 I art     :   at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:886)
06-03 20:41:57.964  9462  9462 I art     :   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:776)
06-03 20:41:57.964  9462  9462 I art     : 
06-03 20:41:57.979  9462  9462 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
06-03 20:41:57.996  9462  9462 V BoostFramework: mAcquireFunc method = public int com.qualcomm.qti.Performance.perfLockAcquire(int,int[])
06-03 20:41:57.997  9462  9462 V BoostFramework: mReleaseFunc method = public int com.qualcomm.qti.Performance.perfLockRelease()
06-03 20:41:57.997  9462  9462 V BoostFramework: mAcquireTouchFunc method = public int com.qualcomm.qti.Performance.perfLockAcquireTouch(android.view.MotionEvent,android.util.DisplayMetrics,int,int[])
06-03 20:41:57.997  9462  9462 V BoostFramework: mIOPStart method = public int com.qualcomm.qti.Performance.perfIOPrefetchStart(int,java.lang.String)
06-03 20:41:57.997  9462  9462 V BoostFramework: mIOPStop method = public int com.qualcomm.qti.Performance.perfIOPrefetchStop()
06-03 20:41:58.001  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@7b65ee2
06-03 20:41:58.001  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@e5b7273
06-03 20:41:58.001  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@1c0f530
06-03 20:41:58.001  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@cfd35a9
06-03 20:41:58.010  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@f0c2cf
06-03 20:41:58.010  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@cf8345c
06-03 20:41:58.012  9462  9462 I ViewConfigCompat: Could not find method getScaledScrollFactor() on ViewConfiguration
06-03 20:41:58.034  9462  9462 D MainActivity: Got intent: Intent { flg=0x10000000 cmp=org.openhab.habdroid/.ui.MainActivity bnds=[834,1219][1056,1344] }
06-03 20:41:58.039  9462  9462 D MainActivity: onStart()
06-03 20:41:58.039  9462  9462 D PageConnectionHolderFragment: onStart(), started false
06-03 20:41:58.040  9462  9462 D MainActivity: onAvailableConnectionChanged()
06-03 20:41:58.040  9462  9462 D ContentController: Update to connection null (message null)
06-03 20:41:58.040  9462  9462 D PageConnectionHolderFragment: updateActiveConnections: URL list [], connection null
06-03 20:41:58.041  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@e31db19
06-03 20:41:58.041  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@a5217de
06-03 20:41:58.052  9462  9462 D MainActivity: onPostCreate()
06-03 20:41:58.053  9462  9462 D MainActivity: onResume()
06-03 20:41:58.057  9462  9462 I am_on_resume_called: [0,org.openhab.habdroid.ui.MainActivity,LAUNCH_ACTIVITY]
06-03 20:41:58.094  9462  9509 I Adreno  : QUALCOMM build                   : aa39516, I97e7dfc697
06-03 20:41:58.094  9462  9509 I Adreno  : Build Date                       : 04/17/17
06-03 20:41:58.094  9462  9509 I Adreno  : OpenGL ES Shader Compiler Version: XE031.09.00.04
06-03 20:41:58.094  9462  9509 I Adreno  : Local Branch                     : 
06-03 20:41:58.094  9462  9509 I Adreno  : Remote Branch                    : quic/LA.BR.1.3.6.c2_rb2.12
06-03 20:41:58.094  9462  9509 I Adreno  : Remote Branch                    : NONE
06-03 20:41:58.094  9462  9509 I Adreno  : Reconstruct Branch               : NOTHING
06-03 20:41:58.100  9462  9509 I OpenGLRenderer: Initialized EGL, version 1.4
06-03 20:41:58.100  9462  9509 D OpenGLRenderer: Swap behavior 1
06-03 20:41:58.131  9462  9462 D MainActivity: onCreateOptionsMenu()
06-03 20:41:58.131  9462  9462 D MainActivity: onPrepareOptionsMenu()
06-03 20:41:58.824  9462  9505 D AbstractConnection: Socket connected (attempt  0)
06-03 20:41:58.824  9462  9505 D ConnectionFactory: Connecting to local URL
06-03 20:41:58.824  9462  9462 D MainActivity: onAvailableConnectionChanged()
06-03 20:41:58.829  9462  9462 D ContentController: Update to connection org.openhab.habdroid.core.connection.DefaultConnection@4685fd8 (message null)
06-03 20:41:58.829  9462  9462 D PageConnectionHolderFragment: updateActiveConnections: URL list [], connection org.openhab.habdroid.core.connection.DefaultConnection@4685fd8
06-03 20:41:58.832  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@7aa8b97
06-03 20:41:58.832  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@7efa184
06-03 20:41:58.842  9462  9462 D MainActivity: onCreateOptionsMenu()
06-03 20:41:58.842  9462  9462 D MainActivity: onPrepareOptionsMenu()
06-03 20:41:59.093  9462  9462 D ServerProperties: Server returned sitemaps: [Sitemap{name=tf, label=Tempus Fugit, link=https://<openhab-local-address>/rest/sitemaps/tf, icon=null, iconPath=icon/null, homepageLink=https://<openhab-local-address>/rest/sitemaps/tf/tf}]
06-03 20:41:59.096  9462  9462 D MainActivity: Configured sitemap is 'tf', selected Sitemap{name=tf, label=Tempus Fugit, link=https://<openhab-local-address>/rest/sitemaps/tf, icon=null, iconPath=icon/null, homepageLink=https://<openhab-local-address>/rest/sitemaps/tf/tf}
06-03 20:41:59.096  9462  9462 I MainActivity: Opening sitemap Sitemap{name=tf, label=Tempus Fugit, link=https://<openhab-local-address>/rest/sitemaps/tf, icon=null, iconPath=icon/null, homepageLink=https://<openhab-local-address>/rest/sitemaps/tf/tf}, currently selected null
06-03 20:41:59.096  9462  9462 D ContentController: Opening sitemap Sitemap{name=tf, label=Tempus Fugit, link=https://<openhab-local-address>/rest/sitemaps/tf, icon=null, iconPath=icon/null, homepageLink=https://<openhab-local-address>/rest/sitemaps/tf/tf}
06-03 20:41:59.097  9462  9462 D PageConnectionHolderFragment: updateActiveConnections: URL list [https://<openhab-local-address>/rest/sitemaps/tf/tf], connection org.openhab.habdroid.core.connection.DefaultConnection@4685fd8
06-03 20:41:59.097  9462  9462 D PageConnectionHolderFragment: Creating new handler for URL https://<openhab-local-address>/rest/sitemaps/tf/tf
06-03 20:41:59.097  9462  9462 D PageConnectionHolderFragment: Creating new SSE helper for sitemap tf, page tf
06-03 20:41:59.097  9462  9462 D PageConnectionHolderFragment: Loading data for https://<openhab-local-address>/rest/sitemaps/tf/tf, long polling false
06-03 20:41:59.103  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@c59486f
06-03 20:41:59.163  9462  9462 D MainActivity: onOptionsItemSelected()
06-03 20:41:59.272  9462  9462 D PageConnectionHolderFragment: Updated page data for URL https://<openhab-local-address>/rest/sitemaps/tf/tf (12 widgets)
06-03 20:41:59.272  9462  9462 D ContentController: Got update for URL https://<openhab-local-address>/rest/sitemaps/tf/tf, pending [https://<openhab-local-address>/rest/sitemaps/tf/tf]
06-03 20:41:59.288  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@7175cae
06-03 20:41:59.288  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@fc3aa4f
06-03 20:41:59.291  9462  9462 D WidgetListFragment: onViewCreated() https://<openhab-local-address>/rest/sitemaps/tf/tf
06-03 20:41:59.291  9462  9462 D WidgetListFragment: onActivityCreated() https://<openhab-local-address>/rest/sitemaps/tf/tf
06-03 20:41:59.292  9462  9462 D WidgetListFragment: onStart() https://<openhab-local-address>/rest/sitemaps/tf/tf
06-03 20:41:59.293  9462  9462 D PageConnectionHolderFragment: Trigger update for URL https://<openhab-local-address>/rest/sitemaps/tf/tf, force false
06-03 20:41:59.293  9462  9462 D ContentController: Got update for URL https://<openhab-local-address>/rest/sitemaps/tf/tf, pending []
06-03 20:41:59.304  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/attic?state=&format=PNG, avoidCache false
06-03 20:41:59.304  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/attic?state=&format=PNG, avoidCache false
06-03 20:41:59.311  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/firstfloor?state=&format=PNG, avoidCache false
06-03 20:41:59.312  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/firstfloor?state=&format=PNG, avoidCache false
06-03 20:41:59.317  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/groundfloor?state=&format=PNG, avoidCache false
06-03 20:41:59.318  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/groundfloor?state=&format=PNG, avoidCache false
06-03 20:41:59.323  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/terrace?state=&format=PNG, avoidCache false
06-03 20:41:59.324  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/terrace?state=&format=PNG, avoidCache false
06-03 20:41:59.334  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/temperature?state=26.54&format=PNG, avoidCache false
06-03 20:41:59.335  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/temperature?state=26.54&format=PNG, avoidCache false
06-03 20:41:59.343  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/calendar?state=2019-06-03T20%3A41%3A46.738%2B0200&format=PNG, avoidCache false
06-03 20:41:59.344  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/calendar?state=2019-06-03T20%3A41%3A46.738%2B0200&format=PNG, avoidCache false
06-03 20:41:59.356  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/settings?state=&format=PNG, avoidCache false
06-03 20:41:59.357  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/settings?state=&format=PNG, avoidCache false
06-03 20:41:59.362  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/lock?state=&format=PNG, avoidCache false
06-03 20:41:59.363  9462  9462 I WidgetImageView: Refreshing image at https://<openhab-local-address>/icon/lock?state=&format=PNG, avoidCache false
06-03 20:42:00.404  9462  9462 D MainActivity: onCloudConnectionChanged()
06-03 20:42:00.421  9462  9539 D FcmRegistrationService: Register device at openHAB-cloud with URL: addAndroidRegistration?deviceId=9a2d1c5dfcb4c1e6&deviceModel=Bq+Aquaris+X5+Plus&regId=fIQBgMkvUG4:APA91bELay_N-4ZBqbKGFRcBoREv__EmH4vXSgKuJt9FmaWD4PEoHGrL2mtOHj6bOxe03D6EfpMzDSjFuYinX-DXWau_-VaN4DgAHC32aKMRsmNW3CGfxbMYGz0eV7S9IwJEzpq5GdpPRdmKILa1bvYV9SV1W0OyDw
06-03 20:42:01.220  9462  9539 D FcmRegistrationService: FCM reg id success
06-03 20:42:01.575  9462  9462 D MainActivity: onPause()
06-03 20:42:01.575  9462  9462 D WidgetListFragment: onPause() https://<openhab-local-address>/rest/sitemaps/tf/tf
06-03 20:42:01.577  9462  9462 I am_on_paused_called: [0,org.openhab.habdroid.ui.MainActivity,handlePauseActivity]
06-03 20:42:01.610  9462  9462 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
06-03 20:42:01.651  9462  9462 E ExtMediaPlayer-JNI: env->IsInstanceOf fails
06-03 20:42:01.651  9462  9462 E MediaPlayer-JNI: JNIMediaPlayerFactory: bIsQCMediaPlayerPresent 0
06-03 20:42:01.651  9462  9462 E ExtMediaPlayer-JNI: env->IsInstanceOf fails
06-03 20:42:01.651  9462  9462 E MediaPlayer-JNI: JNIMediaPlayerFactory: bIsQCMediaPlayerPresent 0
06-03 20:42:01.773  9462  9462 D MediaPlayer: setSubtitleAnchor in MediaPlayer
06-03 20:42:01.779  9462  9462 D Ringtone: Successfully created local player
06-03 20:42:01.796  9462  9462 D PreferencesActivity: Removing notification prefs for >= 25
06-03 20:42:01.814  9462  9462 I am_on_resume_called: [0,org.openhab.habdroid.ui.PreferencesActivity,LAUNCH_ACTIVITY]
06-03 20:42:02.058  9462  9462 D MainActivity: onSaveInstanceState()
06-03 20:42:02.064  9462  9462 D MainActivity: onStop()
06-03 20:42:02.064  9462  9462 D PageConnectionHolderFragment: onStop()
06-03 20:42:02.064  9462  9462 D PageConnectionHolderFragment: Canceling connection for URL https://<openhab-local-address>/rest/sitemaps/tf/tf
06-03 20:42:02.065  9462  9462 I am_on_stop_called: [0,org.openhab.habdroid.ui.MainActivity,handleStopActivity]
06-03 20:42:02.524  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@de7ff45
06-03 20:42:02.525  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@fece69a
06-03 20:42:03.730  9462  9462 I am_on_paused_called: [0,org.openhab.habdroid.ui.PreferencesActivity,handlePauseActivity]
06-03 20:42:03.770  9462  9462 I AppCompatViewInflater: app:theme is now deprecated. Please move to using android:theme instead.
06-03 20:42:03.788  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@2d8649d
06-03 20:42:03.788  9462  9462 V BoostFramework: BoostFramework() : mPerf = com.qualcomm.qti.Performance@338dd12
06-03 20:42:03.799  9462  9462 I am_on_resume_called: [0,org.openhab.habdroid.ui.LogActivity,LAUNCH_ACTIVITY]

```

**openHAB Server log**
not applicable

**Comment by magd1978** [comment]
Sorry for the delay @ maniac103.
Attached file with log generated by MatLog while the error occurred: [logs-2019-06-10-10-30-02.zip](https://github.com/openhab/openhab-android/files/3271254/logs-2019-06-10-10-30-02.zip)
Thank you!

**Comment by maniac103** [comment]
```
06-10 10:29:05.840 E/AndroidRuntime(30967): FATAL EXCEPTION: main
06-10 10:29:05.840 E/AndroidRuntime(30967): Process: com.google.android.googlequicksearchbox:search, PID: 30967
06-10 10:29:05.840 E/AndroidRuntime(30967): java.lang.IllegalStateException: Activity has been destroyed
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at android.app.FragmentManagerImpl.enqueueAction(FragmentManager.java:1456)
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at android.app.BackStackRecord.commitInternal(BackStackRecord.java:708)
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at android.app.BackStackRecord.commitAllowingStateLoss(BackStackRecord.java:676)
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at com.google.android.voicesearch.intentapi.b.a(SourceFile:16)
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at com.google.common.s.a.bh.run(SourceFile:5)
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at com.google.android.apps.gsa.shared.util.c.b.be.run(SourceFile:2)
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at android.os.Handler.handleCallback(Handler.java:751)
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at android.os.Handler.dispatchMessage(Handler.java:95)
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at android.os.Looper.loop(Looper.java:154)
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at android.app.ActivityThread.main(ActivityThread.java:6119)
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at java.lang.reflect.Method.invoke(Native Method)
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
06-10 10:29:05.840 E/AndroidRuntime(30967): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
06-10 10:29:05.842 W/ActivityManager(2765):   Force finishing activity com.google.android.googlequicksearchbox/com.google.android.voicesearch.intentapi.IntentApiActivity
```

This is the actual crash ... seeing that 'activity has been destroyed', did you maybe press back shortly after starting the voice recognition?

**Comment by magd1978** [comment]
> This is the actual crash ... seeing that 'activity has been destroyed', did you maybe press back shortly after starting the voice recognition?
No, I didn't press back shortly after starting the voice recognition.
I simply clicked the voice recognition button on the widget and a pop-up window with Google's error appeared. Sometimes there is a "reload" option and sometimes a "close" option. Once this is done, if you click on the widget again, it works fine. Sometimes the error appears once, and sometimes twice, until the next one voice recognition works.

**Comment by mueller-ma** [comment]
Do you have some overly aggressive battery saver apps running? https://dontkillmyapp.com/

**Comment by magd1978** [comment]
I do not use any battery optimizer, except for the one included in Android Nougat, and I have tried to put the Google and openHab applications in the optimizer settings as not optimized, but the problem has not been solved.
The failure must have occurred after some update of Google services, because I think that about a month ago it did not happen. I have also installed the openHab Beta 2.0.43 app that has always worked well, that has not been updated for more than a year and now also fails. That's why I think it should be a problem with some Google update.

**Comment by magd1978** [comment]
I confirm my suspicion about Google services. I have uninstalled the updates of the "Google" app, and with version 7.17.28.21.arm the voice commands WORK PERFECTLY and without errors.
After this check, I proceeded to update the "Google" app again to the latest available version (10.0.6.21.arm) and the failure occurred again.
I have been able to verify that the problem also occurs with another quite old application that I have. It is probably a problem outside the openHab app that you can not solve from within the app.
Sorry for bothering you.
